server/report/management/commands/clear_reports.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import uuid
+from django.core.management.base import BaseCommand
+from report.models import Report
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Deletes all reports"
+
+    def add_arguments(self, parser):
```

Comment:
Why do we want to delete reports? I think we should archive instead

Reply:
used while testing, probably shouldn't ever be used in prod though